release.conf

```diff
@@ -1,8 +0,0 @@
-[Defaults]
-name = getconf
-with_wheel = 1
-changelog = ChangeLog
-version_path = getconf/__init__.py
```

Comment:
Is this set in `setup.cfg` for zest.releaser?

Reply:
Yes: https://github.com/Polyconseil/getconf/blob/748de35ba2c3f41e8feab1616454a311ad0b3ab1/setup.cfg#L6